gravitee-apim-repository/gravitee-apim-repository-elasticsearch/README.adoc

```diff
@@ -5,8 +5,6 @@
 By default, unit tests are run with a TestContainer version of ElasticSearch 7.17.7, but sometimes it can be useful to run them against other version of ElasticSearch.
 To do so you can use the following commands:
 
-* ES 5.x: `mvn clean test -Delasticsearch.version=5.6.16`
-* ES 6.x: `mvn clean test -Delasticsearch.version=6.8.23`
 * ES 7.x: `mvn clean test -Delasticsearch.version=7.17.8`
```

Comment:
I'm wondering if we could not also get rid of ES 7.

Reply:
I was not sure because I thought ES7 and Opensearch 1 was pretty much the same 🙈

Reply:
I understand. Let's keep ES 7 for the moment. We will eventually remove ES 7 support in a dedicated ticket, as it concerns also the reporters.